onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -773,7 +834,7 @@ def quantize_bias(self, node, new_node_list):
 
         return quantized_bias_name
 
-    def quantize_inputs(self, node, indices, initializer_use_weight_qType=True):
+    def quantize_inputs(self, node, indices, initializer_use_weight_qType=True, symmetric=True, reduce_range=False, per_channel=False, axis=-1):
```

Comment:
>per_channel [](start = 116, length = 11)

the per_channel should be controlled by the self.per_channel parameter. Could you please refer to per-channel in conv [https://github.com/microsoft/onnxruntime/blob/344a2a8ee5b3e8d626a030381b95ca701237c890/onnxruntime/python/tools/quantization/operators/conv.py#L80](https://github.com/microsoft/onnxruntime/blob/344a2a8ee5b3e8d626a030381b95ca701237c890/onnxruntime/python/tools/quantization/operators/conv.py#L80)?

Reply:
@yufenglee I thought `self.per_channel` is more for global control. And, we might want to have different `per_channel` option for each operator. So, in this PR, it uses `per_channel` only if both `self.per_channel` and the parameter `per_channel` are true.

Reply:
One reason for this is embedding matrix has quite different behavior in terms of quantization compared to the other layers. The embedding matrix is used for A and the other weights are used for B.